### PR TITLE
[OSDOCS-12652]: Add latency prereq to HCP support matrix

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-12652

Link to docs preview: https://85341--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-prepare/hcp-requirements.html#hcp-matrix-hc_hcp-requirements

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by lahinson** [comment]
@zanetworker When you can, PTAL. Thanks!

**Review by zanetworker — APPROVED** [review]

**Comment by zanetworker** [comment]
Thanks @lahinson looks good to me!

**Comment by LiangquanLi930** [comment]
lgtm

File: modules/hcp-support-matrix.adoc [hunk 1]
```diff
@@ -35,7 +35,14 @@ Any supported standalone {product-title} cluster can be a management cluster. Th
 [id="hcp-matrix-hc_{context}"]
 == Hosted cluster support
 
-For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version.
```
Comment (mburke5678) [inline modules/hcp-support-matrix.adoc:38]: I know this is exiting text, but there seems to be a word missing here. 
```suggestion
For hosted clusters, no direct relationship exists between the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. 
```

File: modules/hcp-support-matrix.adoc [hunk 2]
```diff
@@ -35,7 +35,14 @@ Any supported standalone {product-title} cluster can be a management cluster. Th
...
 
-For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. 
+
+[NOTE]
+====
+Ensure a minimum latency of 200 ms between the management cluster and hosted clusters.
+====
+
+The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
```
Comment (mburke5678) [inline modules/hcp-support-matrix.adoc:45]:
```suggestion
The following table maps {mce-short} versions to the hosted cluster versions that you can create by using the HyperShift Operator associated with that version of {mce-short}:
```

**Comment by mburke5678** [comment]
@lahinson The new note reads fine. I added a couple of comments about the existing text. Feel free to ignore as out of scope for the PR, as you deem necessary.

File: modules/hcp-support-matrix.adoc (hunk 2 above)
Comment (lahinson) [reply modules/hcp-support-matrix.adoc:45]: Much better, thanks.

**Pull request merged**